.github/workflows/main.yml

```diff
@@ -90,7 +90,10 @@ jobs:
           - TEST=HDFDataset
           - TEST=TranslationDataset
           - TEST=LearningRateControl
+          - TEST=Log
```

Comment:
This test requires TF.

Reply:
Still, the log-test should not be removed just because the TF backend would be removed, but re-written. Thus I would prefer it under "base-tests"

Reply:
What do you mean by removed? Why do you think it should be removed?

I just said it requires TF. So it means it must be under the TF tests.

It cannot be under the base tests when it requires TF.

I don't really see an option to rewrite it as well. It really tests TF specific things. Maybe rename the test to test_TF_Log.py if you like, and then you can have a separate test_Log.py to really only test base things, but I think we don't really have anything for that.

Reply:
> Why do you think it should be removed?

I do not think that. 

> So it means it must be under the TF tests.

I will just move it, then we do not need to discuss.